using_openshift/managing_nodes.adoc

```diff
@@ -0,0 +1,128 @@
+= Managing Nodes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+As an OpenShift administrator, you can manage link:../architecture/kubernetes_infrastructure.html#node[nodes] in your instance using the CLI.
+
+The CLI manages and provides information about known nodes in your instance by interacting with node objects. A node object is a representation of a node, which can be a physical or virtual system. The master uses this information to validate the node with health checks. See the https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/node.md#node-management[Kubernetes documentation] for more information on node management.
+
+== Listing Nodes
+To list all nodes that are known to your OpenShift instance, run `osc get nodes`:
+
+----
+$ osc get nodes
+NAME                 LABELS              STATUS
+node1.example.com    <none>              Ready
+node2.example.com    <none>              NotReady
+----
+
```

Comment:
@adellape Should the above example be in example tags? ATM they're just in code blocks. There's also a few more examples below. If not, maybe some replaceables might be in order in some of the command examples.